src/components/PageSizeSelectList/PageSizeSelectList.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { PaginationOption } from '../../types/PaginationOption';
+import { defaultPaginVal } from '../../utils';
+
+type Props = {
+  selectOptions: number[],
+  paginationOption: PaginationOption,
+  onSetPaginationOption: (value: PaginationOption) => void,
+};
+
+export const PageSizeSelectList: React.FC<Props> = ({
+  selectOptions,
+  paginationOption,
+  onSetPaginationOption,
+}) => {
+  const handleItemPerPage
+  = (event: React.ChangeEvent<HTMLSelectElement>): void => {
+    onSetPaginationOption({
```

Comment:
The same about actual state.

Reply:
Not fixed 😢 
If we want to work with the actual state, it should be received through a callback, and not applied directly.